### PR TITLE
There are no more sign ups for expanded anomaly detection

### PR DESCRIPTION
**Pull request by paperclypse**
As per @rr0214.

<sup> Created from VS Code using [CodeStream](https://codestream.com/?utm_source=cs&utm_medium=pr&utm_campaign=github*com)</sup>

### CODE REVIEW

**Review by barbnewrelic — CHANGES_REQUESTED** [review]
Made a comment via CodeStream. After you fix the typo, feel free to merge. No need for second review from me!

File: src/content/docs/alerts-applied-intelligence/applied-intelligence/proactive-detection/expanded-anomaly-detection.mdx
Comment (paperclypse) [inline src/content/docs/alerts-applied-intelligence/applied-intelligence/proactive-detection/expanded-anomaly-detection.mdx:11]:
```suggestion
  This feature is a limited release and we're no longer accepting requests to sign up.
```

**Pull request merged**
